openapitools.json

```diff
@@ -14,7 +14,7 @@
         "ignoreFileOverride": "./.openapi-generator-ignore",
         "library": "generichost",
         "additionalProperties": {
-          "packageVersion": "3.0.0",
+          "packageVersion": "3.0.1",
```

Comment:
I would probably make this 3.1.0 instead of 3.0.1 because there are some enums that have been removed and deprecated endpoints.  So it seems bigger than a bug fix.